project_close_forecast_date/security/ir.model.access.csv

```diff
@@ -0,0 +1,2 @@
+"id","name","model_id:id","group_id:id","perm_read","perm_write","perm_create","perm_unlink"
+"access_project_close_month_wizard_manager","access.project_close_month_wizard_manager","project_close_forecast_date.model_project_close_month_wizard","project.group_project_manager",1,1,1,1
```

Comment:
This suggests that only Project Managers can execute this new action (or wizard launch will fire an error). In `project_profitability_fields` implementation `last_close_date` can be edited by a Project User. Why this change in such permissions? Should relax this, or require being Project Manager in general for every `last_close_date`?